### ISSUE TITLE
[Accessibility] 1xMissing form label, 2xEmpty button and 1xBroken ARIA reference - Admin Panel -> Modify pop-up

### ISSUE THREAD

**Issue by cmunteanuRW** [Talleh/VolleyballPub#10]
**Steps to reproduce:**

https://github.com/user-attachments/assets/5f21ac12-cca1-4203-a645-ca3a1ca1f106

**Actual Result:**

![Image](https://github.com/user-attachments/assets/8bf507fa-cf8a-47db-ba6d-66077c7651a6)

**Expected Result:**
No accessibility issues are present
